_includes/head.html

```diff
@@ -9,7 +9,7 @@
   <link rel="icon" type="image/png" href="{{ site.url }}/favicon-16x16.png" sizes="16x16">
   <link rel="mask-icon" href="{{ site.url }}/safari-pinned-tab.svg" color="#456BD9">
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700" crossorigin>
-  <link rel="stylesheet" href="https://cloudfour-patterns.netlify.com/assets/toolkit/styles/toolkit.css" crossorigin>
+  <link rel="stylesheet" href="/main.css">
```

Comment:
Is there a service worker or anything we need to update for this?

Reply:
It should be handled by the `assetRoute` section. I removed the section that caches the pattern library.